safe-global/safe-wallet-web
# feat: Display network logos in network selector

## What it solves

- Shows network logos for the network selector
- Splits up the network selector into testnets and non-testnets
- Shows network logos for the chain indicator except in the sidebar and safe list sidebar

## How to test it

1. Open Safe and observe network logos in the network selector
2. Open the sidebar and observe no network logo there
3. Open the safe list sidebar and observe no network logo there

## Screenshots

<img width="225" alt="Screenshot 2023-12-11 at 12 10 29" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/924116a7-3832-493b-ad5b-3f4b106f28a6">
<img width="257" alt="Screenshot 2023-12-11 at 15 34 11" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/44424f0b-5cc4-4964-bce3-1f5fd7df02fc">
<img width="507" alt="Screenshot 2023-12-11 at 15 34 23" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/7f79e323-364b-477f-8b54-b08731ea0886">
<img width="466" alt="Screenshot 2023-12-11 at 15 34 40" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/10fc7069-18b7-44f5-9312-bb4f6562ca14">
<img width="1003" alt="Screenshot 2023-12-11 at 15 34 58" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/4c6ab763-dd72-414c-8352-7a5b7716dbc3">

## Checklist
